apps/console/src/features/groups/components/group-common-components/render-chip.tsx

```diff
@@ -51,12 +54,13 @@ export const RenderChip: FunctionComponent<RenderChipInterface> = (
     const {
         key,
         setActiveOption,
-        primaryText,
-        option,
-        activeOption
+        displayName,
+        audienceType,
+        audienceDisplay,
+        option
     } = props;
 
-    const [ popoverAnchorEl, setPopoverAnchorEl ] = useState<Element>(null);
+    const [ , setPopoverAnchorEl ] = useState<Element>(null);
```

Comment:
Can't we remove this as well?

Reply:
+1

Reply:
Fixed with 8cb8bf2176f03d3406c2b5b4530b56bf77c07a78